docs_src/main/Sidebar.svelte

```diff
@@ -45,18 +15,33 @@
       <p class="flex--grow width-1of1 color--graa1 fontweight-bold">Design system</p>
     </div>
   </div>
-  {#each groupedMenuItems as group}
+  <div class="sidebar-menuitem-container padding-l">
+    <div class="sidebar-submenu-items">
+      <a
+        class="sidebar-item width-1of1 padding-m--t padding-m--rl"
+        class:active-item={'/' === $location}
+        href="#a11y"
+        use:link={{ disabled: false, href: '/' }}
+      >
+        Overview
+      </a>
+    </div>
+  </div>
+  {#each menuItems as group}
     <div class="sidebar-menuitem-container padding-l">
-      {#if group.length && group[0].group}
-        <div class="sidebar-submenu-title fontsize-small">{titleCase(group[0].group)}</div>
-      {/if}
+      <div class="sidebar-submenu-title fontsize-small">
+        <a class:active-item={group.href === $location} href="#a11y" use:link={{ disabled: false, href: group.href }}>
+          {group.title}
+        </a>
+      </div>
+
       <div class="sidebar-submenu-items">
-        {#each group as menuItem}
+        {#each group.routes as menuItem}
           <a
             class="sidebar-item width-1of1 padding-m--t padding-m--rl"
-            class:active-item={menuItem.link === url}
-            href={menuItem.link}
-            use:link
+            class:active-item={menuItem.href === $location}
+            href="#a11y"
```

Comment:
Hvad betyder det her #a11y? :)

Reply:
a11y = accessibility
a 11 (karakterer) y

Den er der for at gøre sveltes accessibility checker glad - den har ikke nogen reel effekt